OloEngine/src/OloEngine/Scripting/VisualScript/VisualScriptNodeRegistry.cpp

```
@@ -0,0 +1,88 @@
+        m_Types.insert_or_assign(std::move(key), std::move(descriptor));
+        m_SortedDirty = true;
+        return true;
+    }
+
+    const NodeTypeDescriptor* NodeRegistry::Find(std::string_view typeName) const
+    {
+        const auto it = m_Types.find(typeName);
+        return it == m_Types.end() ? nullptr : &it->second;
+    }
+
+    const std::vector<const NodeTypeDescriptor*>& NodeRegistry::GetSorted() const
```

Cppcheck (2.21.0)

[style] The function 'GetSorted' is never used.

(unusedFunction)